MediaBrowser.Api/LiveTv/LiveTvService.cs

```diff
@@ -866,10 +868,30 @@ private void AssertUserCanManageLiveTv()
 
         public async Task<object> Post(AddListingProvider request)
         {
+            if (request.Pw != null)
+            {
+                request.Password = GetHashedString(request.Pw);
+            }
+
+            request.Pw = null;
```

Comment:
what is the reason for this?

Reply:
This is largely just a revert of this [change](https://github.com/jellyfin/jellyfin/commit/0f897589ed6349bb3c88919b06861daa80aec1be#diff-d35b45c019e59607f87b33070bb7ea12). I would guess it is to remove the unhashed password as soon as it has been hashed.